test/members_page_test.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+require_relative './test_helper'
+require_relative '../lib/members_page.rb'
+
+describe MembersPage do
+  around { |test| VCR.use_cassette(url.split('/').last, &test) }
+
+  subject do
+    MembersPage.new(response: Scraped::Request.new(url: url).response)
+  end
+
+  describe 'member rows' do
+    let(:url) { 'http://www.parliament.go.tz/mps-list' }
+
+    it 'contains the expected data' do
+      subject.member_rows.first.to_h.must_equal(id:          '14',
```

Comment:
Even though we're not using `scraped-test` directly here, it would still be good to move test data like this out a YAML file.